### PR TITLE
Help command

### PR DESCRIPTION
**Pull request by victor-hugo-dc**

### CODE REVIEW

File: modules/ydn.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from .base import Module
+import requests
+from bs4 import BeautifulSoup
+import random
+
+
+class YDN(Module):
```
Comment (ErikBoesen) [inline modules/ydn.py]: Oh querying the RSS, that's clever. I was planning to do a search thing, but this works pretty well for now.

File: bot.py [hunk 2]
```diff
@@ -154,10 +155,13 @@ def process_message(message):
                     else:
                         reply("No such command.", group_id)
                 else:
+                    """
                     help_string = "--- Help ---"
                     help_string += "\nStatic commands: " + ", ".join([PREFIX + title for title in static_commands])
                     help_string += "\nTools: " + ", ".join([PREFIX + title for title in commands])
                     help_string += f"\n(Run `{PREFIX}help commandname` for in-depth explanations.)"
+                    """
+                    help_string = "https://victor-hugo-dc.github.io/yalebot-help/#help"
```
Comment (ErikBoesen) [inline bot.py]: I would prefer to integrate this into the actual bot website and just serve it through Flask

Comment (ErikBoesen) [reply bot.py]: I'll implement this with your code if you like

File: modules/ydn.py (hunk 1 above)
Comment (victor-hugo-dc) [reply modules/ydn.py]: I updated it to do the search

**Pull request merged**